support/ebpf/Makefile

```diff
@@ -30,11 +34,7 @@ else
 TARGET_FLAGS = -target x86_64-linux-gnu
 endif
 
-ifeq ($(BUILD_TYPE),debug)
-	TARGET_FLAGS+=$(DEBUG_FLAGS)
-endif
-
-FLAGS=$(TARGET_FLAGS) \
+FLAGS=$(TARGET_FLAGS) -g \
```

Comment:
Can we add a comment here as to why we're adding "-g" to the build and later stripping the binary? I'm guessing for BTF.

Reply:
Added a comment with https://github.com/open-telemetry/opentelemetry-ebpf-profiler/pull/560/commits/9ad17e5b0a7486d00f0520790d784cffcea69c4e.